src/components/Charts/ModelChart.js

```diff
@@ -350,38 +362,17 @@ const ModelChart = ({
         }
       >
         <Chart options={options} />
-        {interventions.isCounty && !isEmbed ? (
-          <Disclaimer
-            style={{ border: '2px solid #00d07d', background: 'white' }}
-          >
-            <DisclaimerContent>
-              <b>County data is currently in beta.</b> See something wrong?{' '}
-              <a
-                href="https://forms.gle/NPsLcFnrvfS1kqkn9"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Please let us know.
-              </a>
-            </DisclaimerContent>
-          </Disclaimer>
-        ) : (
-          <Typography></Typography>
-        )}
-        {lastUpdatedDate && (
+        <DisclaimerWrapper>
           <Disclaimer>
-            <DisclaimerContent>
+            <DisclaimerHeader>
               <LightTooltip
                 title="Currently we aggregate data over 4 day intervals to smooth out inconsistencies in the source data. We’re working on improving this now."
                 placement="bottom"
               >
-                <span>
-                  <strong>
-                    Last updated {lastUpdatedDate.toLocaleDateString()}
-                  </strong>
-                  .{' '}
-                </span>
+                <span>Last updated {getDateUpdated()}</span>
```

Comment:
What is the purpose of this?

Reply:
This is to show a valid date that is within three days of today, so we don't have to update the date updated manually.

Reply:
The date is displayed automatically in the current code.